test/util/cart_utils.py

```diff
@@ -261,6 +262,8 @@ def launch_srv_cli_test(self, cartobj, srvcmd, clicmd):
     def launch_test(self, cartobj, cmd, srv1=None, srv2=None):
```

Comment:
unrelated to this review, but i cant figure out what srv1/srv2 arguments here supposed to be

Reply:
They are srv processes. Not exactly the most scalable solution but good enough to get all our tests to avocado since our tests so far utilize up to 2 service groups (group tiers test). Will need to implement a better clean up solution as we scale.